WordPress/Classes/Extensions/Math.swift

```diff
@@ -22,14 +22,13 @@ extension Int {
 }
 
 extension Comparable {
-    /**
-     Clamps self between a minimum and maximum value
-     
-     - returns: the method returns
-        - min if self < min
-        - max if self > max
-        - otherwise it returns self
-     */
+    /// Clamps self between a minimum and maximum value
+    ///
+    /// - Returns: the method returns
+    ///     - min if self < min
+    ///     - max if self > max
+    ///     - otherwise it returns self
+    ///
```

Comment:
Spacing is very important for readability, which is the reason why in my own comments I always add that last blank line.

Even for one liners:

```swift
/// This description is more readable than if you didn't have the spacing.
///
var someExampleVariable: SomeExampleType

/// In this case the variable definition and the comment kind of blend together
var someExampleVariable2: SomeExampleType2
```